server/src/logic/utils/model.utils.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiError, ERRORS } from "./../../models/index";
+
+export class ModelUtils {
```

Comment:
this could be simplified with Object.assign(..):

![bildschirmfoto 2017-03-10 um 18 48 35](https://cloud.githubusercontent.com/assets/6609359/23806527/532eb0f4-05c2-11e7-8bb2-7c1ce31bae0f.png)